// @schibstedspain/sui-card/dist/sui-card/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = SuiCard;

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _classnames = require('classnames');

var _classnames2 = _interopRequireDefault(_classnames);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function SuiCard(props) {
  var classNames = (0, _classnames2.default)(props.className, {
    'sui-Card': !props.className,
    'sui-Card--landscape': props.landscapeLayout,
    'sui-Card--contentfirst': props.landscapeLayout && props.contentFirst
  });

  return _react2.default.createElement(
    'div',
    { className: classNames },
    _react2.default.createElement(
      'div',
      { className: 'sui-Card-primary' },
      props.primary
    ),
    props.secondary && _react2.default.createElement(
      'div',
      { className: 'sui-Card-secondary' },
      props.secondary
    )
  );
}

SuiCard.propTypes = {
  className: _react.PropTypes.string,
  contentFirst: _react.PropTypes.bool,
  landscapeLayout: _react.PropTypes.bool,
  primary: _react.PropTypes.any.isRequired,
  secondary: _react.PropTypes.any
};
